en/lessons/geospatial-data-analysis.md

```diff
@@ -0,0 +1,312 @@
+---
+title: |
+    Using Geospatial Data to Inform Historical Research in R
+    
+date: 2018-08-03
+authors:
+    - Eric Weinberg
+reviewers:
+	 - Lauren Tilton 
```

Comment:
Use 4 spaces when indenting the YAML, instead of using tabs.

Reply:
Thanks. Will do.